server/api/game/controller.js

```diff
@@ -61,11 +63,23 @@ class Game {
 
   endGame = () => {
     clearInterval(this.gameState.interval);
+    this.gameState.status = 2;
   };
 
-  moveGhosts = ({ ghosts, gridState, scatterStart }) => {
+  ifPacmanOnGhost = (ghost) => {
+    const { players } = this.gameState;
+    const { x, y } = ghost;
+    const canConsumeGhost = Object.values(players)
+      .findIndex(player => (player.x === x) && (player.y === y));
+    if (canConsumeGhost !== -1) return true;
```

Comment:
you can just return canConsumeGhost !== -1, it will be automatically evaluated to true/false